Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -869,38 +878,37 @@ ioi ld   (NBCR),a
    inc   hl
    ld    (ix+[_decb]+pd_rx_error),hl
 
-	ld		a,(ix+[_decb]+rcvlsbstat)
+   ; Again, where in the R6000 is this documented?
+#if !defined(_RAB6K)

```

Comment:
Not `NBTESR`.  These are two bytes of the "Receive Status Vector" from "The network port appends six bytes of status information to the last byte of the received data."  See section 25.1 and 25.3 of the [Rabbit 6000 Microprocessor Users Manual](http://ftp1.digi.com/support/documentation/90001108_H.pdf).